cmd/convert.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+var (
+	boolflags = []string{
+		"--debug", "-D",
+		"--verbose",
+		"--log-level",
+		"--l",
+		"--tls",
+		"--tlsverivy",
+	}
+
+	stringflags = []string{
+		"--tlscacert",
+		"--tlscert",
+		"--tlskey",
+		"--host", "-H",
+		"--context",
+	}
+)
+
+func convert(args []string) []string {
+	rootFlags := []string{}
+	command := []string{}
```

Comment:
This differs from the original on `docker/compose-switch` that used to initialise with 
```
command := []string{"compose"}
```

Reply:
doing so, this just breaks the standalone usage :'(

Reply:
and, by the way, we should setup the CI so it run all tests with both `docker compose` and `docker-compose`